sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -93,7 +93,8 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     //region Digital twin APIs
 
     /**
-     * Creates a digital twin.
+     * Creates a digital twin. If the provided digital twin Id is already in use, then this will attempt
```

Comment:
Attempt?

Reply:
Based off of etag, it may fail to replace